### PR TITLE
Create welcome.rst

### PR DESCRIPTION
**Pull request by hillaryfraley**
A rough draft to get us started.

I made my best guesses on link format and deadlines/tickets-on-sale dates based on the Portland content.

### CODE REVIEW

**Comment by plaindocs** [comment]
Hey @hillaryfraley welcome :-)

You can preview that here https://deploy-preview-485--dentist-canary-64055.netlify.com/conf/cincinnati/2018/news/welcome/ (click 'Show all checks', Netlify 'Details', in the green box :arrow_down: : )

Looks like a great start

File: docs/conf/cincinnati/2018/news/welcome.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+.. post:: Apr 18, 2018
+   :tags: cincinnati-2018, website, cfp, tickets, open-help
+
+Announcing Write the Docs Cincinnati + Open Help Website, Call for Organizers, and CFP
+======================================================================================
+
+Welcome to the official launch of `Write the Docs Cincinnati + Open Help 2018 <http://www.writethedocs.org/conf/cincinnati/2018/>`_! We are excited to announce the first-ever collaboration between Write the Docs and Open Help, coming this August.
+
+Since 2011, the Open Help Conference in Cincinnati featured attendee-led discussions and sprints for a wide range of people working on open-source and community-based projects. This year, Write the Docs and Open Help are coming together to bring 5 days of talks, conversations, and doc sprints a little closer to home for our East Coast and Midwestern documentarians!
```
Comment (ericholscher) [inline docs/conf/cincinnati/2018/news/welcome.rst]: 👍

File: docs/conf/cincinnati/2018/news/welcome.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Here's a quick overview of the schedule:
+
+* Day 1: Talks on all things technical documentation
+* Day 2: Unconference discussion and open space
+* Days 3-5: Documentation sprints and writing days
+
+Write the Docs Cincinnati + Open Help is scheduled for August 18-22, 2018 at `Union Hall <http://unionhallcincy.com/>`_ in the historic Over the Rhine neighborhood in `Cincinnati, Ohio <https://www.google.com/maps/place/Union+Hall/@39.109885,-84.515631,15z/data=!4m5!3m4!1s0x0:0x3caee5bfc65866b3!8m2!3d39.109885!4d-84.515631>`_.
+
```
Comment (ericholscher) [inline docs/conf/cincinnati/2018/news/welcome.rst]: Feels like we should probably explain a bit more about the specific events. What is an unconference? What do you need to do to prepare for it? We should be able to spin up the "unconf" page on the website and link to it (similar to http://www.writethedocs.org/conf/portland/2018/unconference/), but it feels like we're doing something "more" with it here. 

@shaunix, do you have existing copy here that we can use here?

**Review by ericholscher — COMMENTED** [review]
This is a great announcement! There isn't too much I'd add except for a bit more explanation and selling of the event format ("Learn, Discuss, Do")

**Comment by plaindocs** [comment]
The only other comment, is do we want to talk a bit more about the WTD+OH collaboration? Or just leave as is?

File: docs/conf/cincinnati/2018/news/welcome.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
+:template: {{year}}/generic.html
+
+.. post:: Apr 18, 2018
+   :tags: cincinnati-2018, website, cfp, tickets, open-help
+
+Announcing Write the Docs Cincinnati + Open Help Website, Call for Organizers, and CFP
+======================================================================================
+
+Welcome to the official launch of `Write the Docs Cincinnati + Open Help 2018 <http://www.writethedocs.org/conf/cincinnati/2018/>`_! We are excited to announce the first-ever collaboration between Write the Docs and Open Help, coming this August.
```
Comment (thatdocslady) [inline docs/conf/cincinnati/2018/news/welcome.rst]: Maybe linkify the WTD and OH first instance (URL for parent community websites)

File: docs/conf/cincinnati/2018/news/welcome.rst (hunk 2 above)
Comment (thatdocslady) [reply docs/conf/cincinnati/2018/news/welcome.rst]: We can link to one of our own Unconf/Writing Day topics (on a related note, we can/should probably have a generic-community version of those that's not linked to a specific conference)

File: docs/conf/cincinnati/2018/news/welcome.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+* Day 1: Talks on all things technical documentation
+* Day 2: Unconference discussion and open space
+* Days 3-5: Documentation sprints and writing days
+
+Write the Docs Cincinnati + Open Help is scheduled for August 18-22, 2018 at `Union Hall <http://unionhallcincy.com/>`_ in the historic Over the Rhine neighborhood in `Cincinnati, Ohio <https://www.google.com/maps/place/Union+Hall/@39.109885,-84.515631,15z/data=!4m5!3m4!1s0x0:0x3caee5bfc65866b3!8m2!3d39.109885!4d-84.515631>`_.
+
+Call for Organizers and Volunteers
+----------------------------------
+
+We need help to make this event a success! If you are interested in volunteering or helping to organize, please email us at cincinnati@writethedocs.org. It's a great opportunity to get involved with the community and also receive a free ticket.
```
Comment (thatdocslady) [inline docs/conf/cincinnati/2018/news/welcome.rst]: we could link to the roles overview of typical conference responsibilities: http://www.writethedocs.org/conf/portland/2018/team/

File: docs/conf/cincinnati/2018/news/welcome.rst [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+Tickets will officially go on sale in **May 2018**.
+
+In the meantime, you can see our 2018 ticket prices on our `Tickets <http://www.writethedocs.org/conf/cincinnati/2018/tickets/>`_ page. As with our other conferences, we have tiered pricing to make it fair for everyone. Please buy the ticket that best fits your situation.
+
+We keep ticket prices low so that it is accessible to the widest range of people possible. However, if you can't afford these prices and still wish to attend, please email us at cincinnati@writethedocs.org, and we can see how we can help.
+
+Stay Updated
+------------
+
+Want to find out what's happening with the conference, or enjoy our monthly global community newsletter?
```
Comment (thatdocslady) [inline docs/conf/cincinnati/2018/news/welcome.rst:57]: should we set up a separate segment in MailChimp for east coast? or just reuse the NA segment?

**Review by thatdocslady — COMMENTED** [review]
Looks great, Hillary! Lots of good info but not overwhelming ;) I second most of the feedback comments and added a few nits/suggestions.

File: docs/conf/cincinnati/2018/news/welcome.rst (hunk 4 above)
Comment (ericholscher) [reply docs/conf/cincinnati/2018/news/welcome.rst]: http://www.writethedocs.org/organizer-guide/confs/event-roles/ is probably the better link.

File: docs/conf/cincinnati/2018/news/welcome.rst (hunk 3 above)
Comment (hillaryfraley) [reply docs/conf/cincinnati/2018/news/welcome.rst]: Thank you -- I added the links. @shaunix looks like the cert for Open Help is still invalid.

File: docs/conf/cincinnati/2018/news/welcome.rst (hunk 2 above)
Comment (hillaryfraley) [reply docs/conf/cincinnati/2018/news/welcome.rst]: Thank you -- I added a little more description for each day. If we have generic pages for each day or conference-specific pages, I can add those links.

File: docs/conf/cincinnati/2018/news/welcome.rst (hunk 4 above)
Comment (hillaryfraley) [reply docs/conf/cincinnati/2018/news/welcome.rst]: Thank you, I added the link.

File: docs/conf/cincinnati/2018/news/welcome.rst (hunk 1 above)
Comment (hillaryfraley) [reply docs/conf/cincinnati/2018/news/welcome.rst]: I added a sentence at the end of the second paragraph to address @ericholscher 's "learn, discuss, do" comment:

"Write the Docs Cincinnati + Open Help 2018 creates a time and place for the community of documentarians to learn, discuss, and do, working together to improve the art and science of documentation."

**Comment by hillaryfraley** [comment]
I just realized that the title cuts off at "Announcing Write the Docs Cincinnati + Open Help Website," on the preview: https://deploy-preview-485--dentist-canary-64055.netlify.com/conf/cincinnati/2018/news/welcome/

**Pull request merged**